src/operations.rs

```diff
@@ -123,24 +123,24 @@ impl WchLink {
         Ok(())
     }
 
-    pub fn write_flash(&mut self, data: &[u8]) -> Result<()> {
+    pub fn write_flash(&mut self, data: &[u8], address: u32) -> Result<()> {
         let pack_size = self.chip.as_ref().unwrap().chip_family.write_pack_size();
         let code_start_addr = self.chip.as_ref().unwrap().memory_start_addr;
-        log::debug!("Code start address 0x{:08x}", code_start_addr);
+        log::debug!("Default flash address 0x{:08x}", code_start_addr);
```

Comment:
Should `code_start_addr` be the default value of address?

Reply:
This line just shows the user the default flash address of the chip. It is basically just an information if the user is unsure what address to use. If you make the address argument optional, you can also use this address.